### ISSUE TITLE
Add a minor release with Composer's replace directive to unblock switchover

### ISSUE THREAD

**Issue by sanmai** [PHPCSStandards/PHP_CodeSniffer#116]
## Is your feature request related to a problem?

[There are many packages](https://packagist.org/packages/squizlabs/php_codesniffer/dependents?order_by=downloads&requires=require) that depend on `squizlabs/php_codesniffer` and I can't install this package because the release version does not include Composer's `replace` directive added in #1.

## Describe the solution you'd like

Please make a minor release such as 3.7.3 including changes from #1. Thanks!

**Comment by jrfnl** [comment]
> I understand the timeline is to push a new release at the end of the week. Yet it may have a breaking change or two, and they could require additional deliberation while the momentum is lost. What I'm proposing it to release `3.7.3` now at the peak of momentum, and let folks update like the normally would to `3.8.0`.

@sanmai I'll keep it in mind. There are still some open questions about the `replace` directive at this moment (also see #113 and #114), so I'll check what's in the changelog before release and will decide then.

**Comment by derrabus** [comment]
All of you, please give @jrfnl some time. Until a new release is tagged from this repository, you won't gain anything from switching just yet.

**Comment by arokettu** [comment]
I have a small suggestion here

the replace directive was added as

```json
    "replace": {
        "squizlabs/php_codesniffer": "> 2.0"
    },
```

shouldn't it be

```json
    "replace": {
        "squizlabs/php_codesniffer": "self.version"
    },
```

?

**Comment by derrabus** [comment]
@arokettu see #113

**Comment by jrfnl** [comment]
Looks like this may not be needed after all - See #135

**Comment by jrfnl** [comment]
Thank you everyone for pitching in, the package rename has been reverted. The package will continue under its original name, so this is no longer relevant.
